packages/L10n/package.json

```diff
@@ -27,5 +27,8 @@
     "react": "^16.8.2",
     "react-dom": "^16.8.2"
   },
-  "gitHead": "8bff2e8a6f01708ba60d9178f867dba378806370"
+  "gitHead": "8bff2e8a6f01708ba60d9178f867dba378806370",
+  "devDependencies": {
```

Comment:
why is this an devDependencie?

Reply:
Because I am just using it in a story

Reply:
I think this should be in the root package.json's devDependencies, we cannot run storybook for only this component anyway